src/Runner/TestSuiteSorter.php

```diff
@@ -51,7 +51,7 @@
     ];
 
     /**
-     * @var array
+     * @var array[string]int Associative array of (string => DEFECT_SORT_WEIGHT) elements
```

Comment:
`array[string]int` is not proper type according to phpdoc
array<string, int>`

Reply:
Thanks! Where did you find this syntax? I saw this recommended a few times, but have found no definitive answer in the manual https://docs.phpdoc.org/guides/types.html

I found some discussions regarding proposals but most of them seem to have gone to sleep, for example: https://github.com/phpDocumentor/phpDocumentor2/issues/650

Reply:
psr-5 always-proposal, sadly. but it's widely suported by IDEs and SCAs